hpedockerplugin/cmd/cmd_createfpg.py

```diff
@@ -0,0 +1,70 @@
+import six
+from oslo_log import log as logging
+
+from hpedockerplugin.cmd import cmd
+from hpedockerplugin import exception
+
+
+LOG = logging.getLogger(__name__)
+FPG_SIZE = 64
+
+
+class CreateFpgCmd(cmd.Cmd):
+    def __init__(self, file_mgr, cpg_name, fpg_name, set_default_fpg=False):
+        self._file_mgr = file_mgr
+        self._fp_etcd = file_mgr.get_file_etcd()
+        self._mediator = file_mgr.get_mediator()
+        self._backend = file_mgr.get_backend()
+        self._cpg_name = cpg_name
+        self._fpg_name = fpg_name
+        self._set_default_fpg = set_default_fpg
+
+    def execute(self):
+        with self._fp_etcd.get_fpg_lock(self._backend, self._fpg_name):
+            self._mediator.create_fpg(self._cpg_name, self._fpg_name)
```

Comment:
`LOG.info` at this place is helpful for indicating to user `fpg` create operation is attempted.